android/layer/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
         }
         externalNativeBuild {
             cmake {
-                cppFlags "-fexceptions", "-std=c++14", "-Wno-nullability-completeness"
+                cppFlags "-fexceptions", "-std=c++17", "-Wno-nullability-completeness"
```

Comment:
fyi @bradgrantham - in this PR, I'm adjusting our android c++ dialect to c++17, matching the other configs.

Reply:
Thanks for doing this.  I think it's important for us to try to maintain consistency between the two.  Especially as we add C++17 features into GFXR.